addons/block_code/ui/blocks/parameter_block/parameter_block.gd

```diff
@@ -35,7 +35,7 @@ func _on_drag_drop_area_mouse_down():
 
 func get_serialized_props() -> Array:
 	var props := super()
-	props.append_array(serialize_props(["block_format", "statement"]))
+	props.append_array(serialize_props(["block_format", "statement", "variant_type"]))
```

Comment:
Oh good catch!

Reply:
Found this in my ParameterInput PR too, hah.